opentelemetry-api/tests/context/propagation/test_tracecontexthttptextformat.py

```diff
@@ -44,60 +44,6 @@ def test_no_traceparent_header(self):
         span_context = FORMAT.extract(get_as_list, output)
         self.assertTrue(isinstance(span_context, trace.SpanContext))
 
-    def test_from_headers_tracestate_entry_limit(self):
```

Comment:
these tests were invalid behaviors in reference to the spec. I removed them as the integration test will validate it is working as intended.